translate/cloud-client/hybrid_glossaries/hybrid_tutorial.py

```diff
@@ -0,0 +1,257 @@
+    # project_id: GCP project id
+    # glossary_name: name you want to give this glossary resource
+    # glossary_uri: the uri of the glossary you uploaded to Cloud Storage
+    #
+    # RETURNS
+    # nothing if glossary creation is successful
+
+    # Instantiates a client
+    client = translate.TranslationServiceClient()
+
+    # Defines the languages in the glossary
+    # This list must match the languages in the glossary
+    #   Here, the glossary includes French and English
+    languages = ['fr', 'en']
+    # Set information to access
+    glossary_uri = 'gs://cloud-samples-data/translation/bistro_glossary.csv'
+    # Designates the data center location that you want to use
+    location = 'us-central1'
+
+    # Set glossary resource name
+    name = client.glossary_path(
+        project_id,
+        location,
+        glossary_name)
+
+    # Set language codes
+    language_codes_set = translate.types.Glossary.LanguageCodesSet(
+        language_codes=languages)
+
+    gcs_source = translate.types.GcsSource(
+        input_uri=glossary_uri)
+
+    input_config = translate.types.GlossaryInputConfig(
+        gcs_source=gcs_source)
+
+    # Set glossary resource information
+    glossary = translate.types.Glossary(
+        name=name,
+        language_codes_set=language_codes_set,
+        input_config=input_config)
+
+    resource = client.location_path(project_id, location)
+
+    # Create glossary resource
+    operation = client.create_glossary(parent=resource, glossary=glossary)
+
+    return operation.result(timeout=90)
+
+
+# [START hybrid_translate]
+def translate_text(text, prev_lang, new_lang, project_id, glossary_name):
+    # Translates text to a given language using a glossary
+    #
+    # ARGS
+    # text: String of text to translate
+    # prev_lang: language of input text
+    # new_lang: language of output text
+    # project_id: GCP project id
+    # glossary_name: name you gave your project's glossary
+    #  resource when you created it
+    #
+    # RETURNS
+    # String of translated text
+
+    # Instantiates a client
+    client = translate.TranslationServiceClient()
+
+    # Designates the data center location that you want to use
+    location = 'us-central1'
+
+    glossary = client.glossary_path(
+        project_id,
+        location,
+        glossary_name)
+
+    glossary_config = translate.types.TranslateTextGlossaryConfig(
+        glossary=glossary)
+
+    resource = client.location_path(project_id, location)
+
+    result = client.translate_text(
+        parent=resource,
+        contents=[text],
+        mime_type='text/plain',  # mime types: text/plain, text/html
+        source_language_code=prev_lang,
+        target_language_code=new_lang,
+        glossary_config=glossary_config)
+
+    # Returns translated text
+    return result.glossary_translations[0].translated_text
+    # [END hybrid_translate]
+
+
+# [START hybrid_glossary_delete]
+def delete_glossary(project_id, glossary_name):
```

Comment:
This method is never called or used.

Reply:
It's necessary for the tests. Would you recommend that I just move the method to the tests file?

Reply:
Yea, if not used in the code, move it to the test code. You can just insert the contents of the method directly into the test that calls it.